dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/logger/LogcatLogHandler.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2019 Datadog, Inc.
+ */
+
+package com.datadog.android.log.internal.logger
+
+import android.os.Build
+import android.util.Log
+
+internal class LogcatLogHandler(
+    internal val serviceName: String
+) : LogHandler {
+
+    // region LogHandler
+
+    override fun handleLog(
+        level: Int,
+        message: String,
+        throwable: Throwable?,
+        attributes: Map<String, Any?>,
+        tags: Set<String>
+    ) {
+        if (Build.MODEL == null) {
+            println("${levelPrefixes[level]}/$serviceName: $message")
+            throwable?.printStackTrace()
+        } else {
+            Log.println(level, serviceName, message)
+            if (throwable != null) {
```

Comment:
isn't rather `if (throwable !=null){
Log.println(.....
} 
else{
Log.println(...
}`
?

Reply:
In both case we log the message. And only if the throwable is not null, we log its stack trace

Reply:
I have the feeling this will look weird in the Logcat because in case you have a Throwable this log will be displayed 2 times in a row (one with and one without the stacktrace). That is the reason why I am thinking that we should better trigger it only once by doing the `if{} else {}`. You will still log the message in both cases but in case you have a throwable you will not log it 2 times in a row.